Flipper/Packages/UI/Sources/Hub/Apps/Screenshot/FullScreenshotsView.swift

```diff
@@ -89,16 +87,11 @@ struct FullScreenshotsView: View {
                         .frame(height: geometry.size.height)
                         .frame(minWidth: geometry.size.width)
                     }
-                    .onChange(of: currentIndex) { newIndex in
+                    .onAppearOrChange(of: currentIndex) { newIndex in
```

Comment:
please don't combine modifiers, just move the code to separate function with index argument